lib/pages/portal/service/website_provider.dart

```diff
@@ -206,7 +192,11 @@ class WebsiteProvider with ChangeNotifier {
             .map((doc) => WebsiteExtension.fromSnap(doc, ownerUid: uid)));
       }
 
-      initializeNumberOfVisits(websites);
+      final bool initializeReturn =
+          await initializeNumberOfVisits(websites, uid);
+      if (!initializeReturn) {
+        AppToast.show('Initialize number of visits failed!');
```

Comment:
This will show in the UI, so it needs to be localized. Make a localized string like "Could not read favourite websites."